x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -115,7 +115,8 @@ export class HeadlessChromiumDriverFactory {
 
         logger.debug(`Browser page driver created`);
       } catch (err) {
-        observer.error(new Error(`Error spawning Chromium browser: [${err}]`));
+        observer.error(new Error(`Error spawning Chromium browser!`));
+        observer.error(err);
```

Comment:
Stringifying error objects in a message has shown to cause problems where the error context or error data gets lost. I searched the code to see if this is happening in other places and fixed them in this PR.